jobs/octopus-energy-python-django-engineer.html

```diff
@@ -0,0 +1,73 @@
+---
+title: Python / Django Engineer
+company: Octopus Energy
+url: https://octopus.energy
+location: Sydney, Australia
+contract: permanent
+contact:
+    name: James Beith
+    email: talent@octopus.energy
+created: !!timestamp 2019-06-05
+tags:
+  - django
+  - python
+  - sydney
+  - australia
+  - postgresql
+  - aws
+  - celery
+  - rabbitmq
+  - terraform
+  - packer
+  - react
+---
+
+We are a tech-focused energy supplier and we're looking for diligent software engineers to join us.
+
+There's a great opportunity for disruption in the energy market at the moment. The big suppliers still dominate, but are not exactly popular. The energy landscape is changing as we move to more disaggregated forms of generation, with less predictability, more dynamism and smarter technology.
+
+You'll be working for a company that is helping address climate change. We're helping move people onto using greener technology and consuming energy from renewable sources. This will all help transition us towards a lower carbon future.
+
+This role offers a great opportunity to help shape the development of Australian market specific functionality into our global energy platform.
+
+# Tech
+
+We're building a modern, event-driven infrastructure for interacting with both consumers (via the web, mobile apps and smart-meters) and the industry (e.g. data flows, consumption forecasting, trading on the wholesale market).
+
+We use Python heavily: our consumer-facing site is powered by Django and the Django-REST-framework amongst other things. We also use Pandas, Numpy and Jupyter for analysis and forecasting, and Celery for background tasks. Client-side, we use React and React-native.
+
+We’re hosted on AWS, and employ most of the Hashistack (Vagrant, Packer, Consul, Terraform) in some form. We employ CircleCI and Atlas to coordinate a continuous deployment pipeline.
+
+Engineering standards are high. We pair program, thoroughly review and test code before it deploys and maintain fast, high-value test suites. We have weekly tech talks, monthly blitz-days (e.g. sharpening our internal tools, upgrading dependencies) and can often be found at tech meet-ups and conferences.
+
+Here's some sample documents:
+
+- Our in-house conventions and styles: [https://github.com/octoenergy/conventions](https://github.com/octoenergy/conventions)
```

Comment:
You might want to format the links so they are clickable, please refer to this documentation - [Markdown Links](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links)